PDF_UA/2/8.10 Forms/8.10.3 Fields/8.10.3.3 Text fields/verapdf-profile-8-10-3-3-t01.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.10 Forms - 8.10.3.3 Text fields - RV and V entries</name>
+        <description>For text fields, when an RV entry is present a V entry shall also be present, and they shall be
+            textually equivalent</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDTextField">
+            <id specification="ISO_14289_2" clause="8.10.3.3" testNumber="1"/>
+            <description>For text fields, when an RV entry is present a V entry shall also be present, and they shall be
+                textually equivalent</description>
+            <test>containsRV == false || (V != null &amp;&amp; RV == V)</test>
+            <error>
+                <message>RV entry is present, but V entry is not present or they are not textually equivalent
```

Comment:
Text field contains RV entry, but no V entry or they are not textually equivalent